.travis/run_tests.sh

```diff
@@ -21,7 +21,7 @@ if [ ! -z ${changed_files} ]; then
 		if [ ! -f ${f} ]; then
 			continue
 		fi
-		ts_name=$(grep -oP 'class[\s]+(\K\w+)(?=\(.*\):)' ${f})
+		ts_name=$(grep -oP 'class[\s]+(\KTest\w+)(?=\(.*\):)' ${f})
```

Comment:
@arungrover I have changed code to ignore \_\_init__.py, please have a look and let me know your view.